code/modules/mob/living/simple_animal/hostile/megafauna/megafauna.dm

```diff
@@ -120,6 +120,9 @@
 		if(L.stat != DEAD)
 			if(!client && ranged && ranged_cooldown <= world.time)
 				OpenFire()
+
+			if(L.health <= HEALTH_THRESHOLD_DEAD && HAS_TRAIT(L, TRAIT_NODEATH)) //Nope, it still gibs yall
```

Comment:
Does gib() go through TRAIT_NODEATH?

Reply:
Yes, anything that calls the death() proc directly does